app/models/pipeline_run.rb

```diff
@@ -299,18 +299,34 @@ def db_load_ercc_counts
   end
 
   def db_load_amr_counts
-    amr_results = PipelineRun.download_file(s3_file_for("amr_counts"), local_amr_full_results_path)
+    amr_drug_mappings = {}
```

Comment:
You should make this a constant AMR_DRUG_MAPPINGS in the AmrCount model. Also, you can define it as follows: 
AMR_DRUG_MAPPINGS = { "AGly": "aminoglycosides", "Bla": "beta-lactamases", ... }